ethportal-peertest/src/constants.rs

```diff
@@ -1,3 +1,46 @@
+use ethportal_api::{HistoryContentKey, HistoryContentValue};
+use serde_json::json;
+
+/// History HeaderWithProof content key & value
+/// Block #1000010
+pub fn test_item() -> (HistoryContentKey, HistoryContentValue) {
+    let content_key: HistoryContentKey =
+        serde_json::from_value(json!(HISTORY_CONTENT_KEY)).unwrap();
+    let content_value: HistoryContentValue =
+        serde_json::from_value(json!(HISTORY_CONTENT_VALUE)).unwrap();
+    (content_key, content_value)
+}
```

Comment:
nitpick ``test_item()`` isn't that descriptive of a name. I have 2 ideas which could improve readability.

- use ``test_header_with_proof()`` instead and remove ``test_item()`` since they are both pre-merge header with proofs so maybe we no longer need ``test_item()``?
- try to use a more descriptive function name maybe like ``test_header_with_proof_block_1000010()`` it is a little long but readability is very nice.

This is a nitpick so if you want to leave it how it is already that is fine